src/Native/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -58,6 +61,28 @@ bool InitializeSystemInfo()
 
 extern bool PalQueryProcessorTopology();
 
+void __stdcall ThreadShutdown();
+
+// Callback that is called when a fiber is detached from a thread
+// Only the home fiber of a thread has the thread pointer stored
+// in the FLS value, others have NULL there.
+void __stdcall FiberDetachCallback(void* lpFlsData)
+{
+    UNREFERENCED_PARAMETER(lpFlsData);
```

Comment:
This is not needed anymore. There used to be just the assert before using it, so that's why it was there to prevent warnings in release builds.